src/der.rs

```diff
@@ -242,9 +242,14 @@ const LONG_FORM_LEN_THREE_BYTES_MAX: usize = (1 << (8 * 3)) - 1;
 // Leading octet for long definite form DER length expressed in subsequent four bytes.
 const LONG_FORM_LEN_FOUR_BYTES: u8 = 0x84;
 
+#[cfg(target_pointer_width = "64")]
 // Maximum size that can be expressed in a four byte long form der len.
 const LONG_FORM_LEN_FOUR_BYTES_MAX: usize = (1 << (8 * 4)) - 1;
 
+#[cfg(target_pointer_width = "32")]
+// Maximum size that can be expressed on 32 bit platforms is the usize limit.
+const LONG_FORM_LEN_FOUR_BYTES_MAX: usize = usize::MAX;
```

Comment:
I think it's more clear to just write `0xff_ff_ff_ff` which gets rid of the need for `target_pointer_width`  too, because that constant doesn't involve a computation larger than `2**32`. Could also write the constants above as `0xff`, `0xff_ff`, `0xff_ff_ff` for consistency?

Reply:
That is much clearer, thank you :+1: Implemented in my latest force push.